packages/styled-base/types/index.d.ts

```diff
@@ -39,7 +39,7 @@ export interface StyledOptions {
 }
 
 export interface StyledComponent<InnerProps, StyleProps, Theme extends object>
-  extends React.SFC<InnerProps & StyleProps & { theme?: Theme }>,
+  extends React.SFC<InnerProps & Omit<StyleProps, 'theme'> & { theme?: Theme }>,
```

Comment:
Could you add a failing test for this?

Reply:
I'm not sure how to ensure this is tested against 3.8. My suspicion is that it isn't and that's why this wasn't caught. But if you could point me to test this against 3.8, I can ensure that a test is added that fails without this change.